images/base/entrypoint

```diff
@@ -61,8 +61,10 @@ fix_machine_id() {
 fix_product_name() {
   # this is a small fix to hide the underlying hardware and fix issue #426
   # https://github.com/kubernetes-sigs/kind/issues/426
-  echo "kind" > /kind/product_name
-  mount -o ro,bind /kind/product_name /sys/class/dmi/id/product_name
+  if [[ $(cat /sys/class/dmi/id/product_name) =~ Google ]]; then
```

Comment:
We should just check if the file exists, I'd prefer not to branch at all in the entry point for a consistent environment.

Reply:
It might even make sense to ensure this directory and fake the whole thing in all environments, but that's a bigger & trickier PR.

Ideally we want kind to act very much the same everywhere.

Reply:
```suggestion
  if [[ ! -f /sys/class/dmi/id/product_name ]]; then
```